Cargo.toml

```diff
@@ -19,19 +19,22 @@ maintenance = { status = "actively-developed" }
 
 [dependencies]
 
-async-stream = "^0"
+#async-stream = "^0"
```

Comment:
Why did you drop this and `tokio-timer` below?

Reply:
was used before, but it was not longer used.